src/Microsoft.Identity.Web.TokenAcquisition/OptionsMergers/ConfidentialClientApplicationOptionsMerger.cs

```diff
@@ -16,7 +16,7 @@ public ConfidentialClientApplicationOptionsMerger(IMergedOptionsStore mergedOpti
         private readonly IMergedOptionsStore _mergedOptionsMonitor;
 
         public void PostConfigure(
-#if NET7_0_OR_GREATER
+#if NET6_0_OR_GREATER
```

Comment:
there are GitHub action warnings which need to be resolved before merging (they are viewable in the GitHub UI)

Reply:
I don't think this change is correct.